src/ui/ResultList/ResultList.ts

```diff
@@ -508,6 +476,15 @@ export class ResultList extends Component {
     return this.fetchingMoreResults;
   }
 
+  public get templateToHtml() {
```

Comment:
Should this be private instead?

I can't think of any legitimate reason why a user would want to call this method directly.